### ISSUE TITLE
[BITV] 9.4.1.2/3.6 - All checkboxes in the first column of the table use the "aria-label" attribute, which has been incorrectly applied in this case on the "span" element, which is not allowed. (2)

### ISSUE THREAD

**Issue by JuliaKirschenheuter** [nextcloud/server#41894]
The "aria-label" attribute should be used together with the "input" element or, alternatively, an existing "label" element should be used instead of ARIA. 

![image](https://github.com/nextcloud/server/assets/6078378/848c7b30-b9b4-4d95-862b-6213ed101453)

https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v3-n6

9.4.1.2/11.6 - https://report.bitvtest.de/default-en/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-bfbf07e5c3-v11-n6

**Comment by susnux** [comment]
Waiting for https://github.com/nextcloud-libraries/nextcloud-vue/pull/4903

**Comment by susnux** [comment]
Fixed with 6dc5ae32b3136a514c26cf0ccc0238072fcbd2b2
